Tests/UIKitTests/UIViewControllerExtensionsTests.swift

```diff
@@ -54,5 +54,15 @@ final class UIViewControllerExtensionsTests: XCTestCase {
         NotificationCenter.default.post(name: notificationIdentifier, object: nil)
         XCTAssertFalse(viewController.notificationFired)
     }
+    
+    func testShowAlert() {
+        let viewController = UIViewController()
+        let title = "test title"
+        let message = "test message"
+        let alertController = viewController.showAlert(title: title, message: message, buttonTitles: ["OK", "Cancel"], highlightedButtonIndex: 1, completion: nil)
```

Comment:
How about testing these as well?
- `buttonTitles`
- `highlightedButtonIndex`
- `completion`
Bonus points if you figure out how to test if the controller was actually displayed 😉

Reply:
Updated test case to include buttonTitles and highlighted buttonIndex. Not sure if we can test completion and whether the view controller was actually presented since that requires user interaction.